dlc-test/src/test/scala/org/bitcoins/dlc/DLCClientTest.scala

```diff
@@ -337,4 +336,187 @@ class DLCClientTest extends BitcoinSAsyncTest {
 
     Future.sequence(testFs).flatMap(_ => executeRefundCase(numOutcomes))
   }
+
+  private def flipAtIndex(bytes: ByteVector, byteIndex: Int): ByteVector = {
```

Comment:
Can we move these functions to somewhere in the `testkit`, that way we can use them in other places. These would be useful for testing the wallet is correctly handling invalid sigs as well